app/src/ui/merge-conflicts/merge-conflicts-warning.tsx

```diff
@@ -12,10 +12,16 @@ interface IMergeConflictsWarningProps {
   readonly onDismissed: () => void
 }
 
+/**
+ * Modal to tell the user their merge encountered conflicts
+ */
 export class MergeConflictsWarning extends React.Component<
   IMergeConflictsWarningProps,
   {}
 > {
+  /**
+   *  displays the repository changes tab and dismisses the modal
```

Comment:
:art: Ensure the new jsdoc entries are `Sentence case™`

Reply:
```suggestion
   *  Displays the repository changes tab and dismisses the modal
```